src/controllers/controller.bar.js

```diff
@@ -125,6 +125,57 @@ function computeFlexCategoryTraits(index, ruler, options) {
 	};
 }
 
+function parseFloatBar(arr, item, vScale, i) {
+	var startValue = vScale._parse(arr[0], i);
+	var endValue = vScale._parse(arr[1], i);
+	var min = Math.min(startValue, endValue);
+	var max = Math.max(startValue, endValue);
+	var barStart = min;
+	var barEnd = max;
+
+	if (Math.abs(min) > Math.abs(max)) {
+		barStart = max;
+		barEnd = min;
+	}
+
+	// Store `barEnd` (furthest away from origin) as parsed value,
+	// to make stacking straight forward
+	item[vScale.id] = barEnd;
+
+	item._custom = {
```

Comment:
Maybe we could compute this at runtime or store in a separate datastructure in the controller so that we don't need to modify by adding `_custom`? In the case that the user passes in data in format we prefer, it'd be nice not to modify their data

Reply:
At the moment, this is internal data - but I see your point.